physionet-django/events/models.py

```diff
@@ -61,3 +64,79 @@ class Meta:
 
     def __str__(self):
         return self.user.get_full_name()
+
+
+class EventApplications(models.Model):
+    """
+    Captures information about applications for events.
+    """
+    class EventApplicationStatus(models.IntegerChoices):
+        WAITLISTED = 0
```

Comment:
Wouldn't text choices be a better choice here, since the integer carries no meaning? e.g. see: https://stackoverflow.com/questions/54802616/how-can-one-use-enums-as-a-choice-field-in-a-django-model

e.g. something like:

```
    class EventApplicationStatus(models.TextChoices):
        WAITLISTED = 'WL', _('Waitlisted')
        APPROVED = 'AP', _('Approved')
        NOT_APPROVED = 'NA', _('Not Approved')
        WITHDRAWN = 'WD', _('Withdrawn')
```

Reply:
hmm. thats true, integer here has no significance. working on this asap